app.js

```diff
@@ -54,10 +55,12 @@ class App {
         const newConfIds = confIds.filter(id => !(id in this.conferenceStates));
         const removedConfIds = Object.keys(this.conferenceStates).filter(id => confIds.indexOf(id) === -1)
         newConfIds.forEach(newConfId => {
-            const dumpId = uuidv4();
+            const clientId = uuidv4();
```

Comment:
This is especially confusing with the name `clientId` as clients _do_ have specific IDs that are generated inside of the bridge.

Reply:
Understood,  should we maybe change it to something like `statsSessionId` so we don't tie the API logic to an actual dump file.

Reply:
Somewhat related question: I think we want to be able to combine multiple dumps for a client who re-joins a call multiple times...did you have anything in mind for how we'd do that?  I only ask because when I read `statsSessionId` it made me think of something which might be on the scope of a single client's entire time in a single meeting (that is, made up of potentially multiple `dumpId`s), but maybe there's a good way to do this without using a separate ID?  If so, then `statsSessionId` sounds fine.

Reply:
Well as it stands, the reconnect mechanism works on a per statsSessionId basis, in case the websocket closes for some reason the client can reconnect using the same statsSessionId, the server side will react to this by creating multiple dumps which can be associated via the statsSessionId.
E.g. 
1. client connects using statsSessionId foo-bar, sends stats, dump foo-bar is created on the server side.
2. websocket disconnects due to some network issue, server reacts by closing the dump, (checks if same statsSessionId was already pushed) pushes associated metadata to dynamo
3. client reconnects using the same statsSessionId, sends stats, dump foo-bar is created again on the server side.
4. client sends close, server checks is statsSessionId was already pushed, it was, adds an increment at the end of the name, pushes foo-bar-1.tar.gz to s3, and metadata with a baseStatsSessionId: foo-bar to dynamo
So we'll end up with multiple dumps foo-bar.tar.gz, foo-bar-1.tar.gz etc. with associated metadata in dynamo, that can be associated in the visualiser.

Reply:
In other words it's up to the client to decide how long he wants a session to last. We can potentially add client logic that keeps the statsSessionId stored for a period of time and whenever he connects use that one instead of generating a new one.

Reply:
Ok sure, if it works on the backend to just have the stats-pusher use the same ID then that works fine.